oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsTransWorker.java

```diff
@@ -36,28 +32,12 @@
     private final MetricsPersistentWorker hourPersistenceWorker;
     private final MetricsPersistentWorker dayPersistenceWorker;
 
-    private final CounterMetrics aggregationHourCounter;
-    private final CounterMetrics aggregationDayCounter;
-
     public MetricsTransWorker(ModuleDefineHolder moduleDefineHolder,
-                              String modelName,
                               MetricsPersistentWorker hourPersistenceWorker,
                               MetricsPersistentWorker dayPersistenceWorker) {
         super(moduleDefineHolder);
         this.hourPersistenceWorker = hourPersistenceWorker;
         this.dayPersistenceWorker = dayPersistenceWorker;
-
-        MetricsCreator metricsCreator = moduleDefineHolder.find(TelemetryModule.NAME)
-                                                          .provider()
-                                                          .getService(MetricsCreator.class);
-        aggregationHourCounter = metricsCreator.createCounter(
-            "metrics_aggregation", "The number of rows in aggregation", new MetricsTag.Keys("metricName", "level",
-                                                                                            "dimensionality"
-            ), new MetricsTag.Values(modelName, "2", "hour"));
-        aggregationDayCounter = metricsCreator.createCounter(
```

Comment:
Why remove all these? The downsampling is important.

Reply:
I didn't remove it. I used model.getDownsampling().getName() in MetricsPersistentWorker class.
The metrics of minutes,hours and days can be exposed.
```
aggregationCounter = metricsCreator.createCounter(
                "metrics_aggregation", "The number of rows in aggregation",
                new MetricsTag.Keys("metricName", "level", "dimensionality"), new MetricsTag.Values(model.getName(), "2", model.getDownsampling().getName())
        );
```